linalg/hypre_parcsr.cpp

```diff
@@ -1697,7 +1697,8 @@ void hypre_ParCSRMatrixBooleanMatvecT(hypre_ParCSRMatrix *A,
 
    if (num_cols_offd)
    {
-#if MFEM_HYPRE_VERSION >= 21100
+#if MFEM_HYPRE_VERSION >= 21100 && 0
```

Comment:
Not sure about the `&& 0`. I don't know the code much.

Reply:
That is on us, we are turning it off since there is some memory management stuff to sort out.